### ISSUE TITLE
minor latex warnings from lineno and fixltx2e

### ISSUE THREAD

**Issue by CrumpLab** [crsh/papaja#336]
Knitting the papaja template gives 2 latex warnings. One of them is:

```
LaTeX Warning: Command \@parboxrestore  has changed.
```

**Expected behavior** is no latex warnings

**Screenshots** of both warnings are below:
<img width="498" alt="Screen Shot 2019-12-19 at 2 06 30 PM" src="https://user-images.githubusercontent.com/4038982/71201813-ce38e600-2268-11ea-930d-5223a5dbc3ea.png">

**possible fix**
It seems the first warning might be addressed by placing:
```
\usepackage{lineno}
```
Early in the list of loaded packages. I put it near the top in a test and compiled without this `\@parboxrestore` warning.

The other warning suggests that the following may no longer be necessary in the .tex template.
```
\usepackage{fixltx2e} % provides \textsubscript
```

**Comment by crsh** [comment]
Thanks, I will implement these changes shortly.

**Comment by crsh** [comment]
I fixed the first warning by loading `csquotes` after `lineno` ([SO](https://tex.stackexchange.com/questions/447006/lineno-package-in-latex-causes-warning-message)). `fixltx2e` is loaded by `pandoc`'s default LaTeX template, which I patch, but was [removed](https://github.com/jgm/pandoc/commit/316083698379ab7a9d14c4010e8c212e89cf720f#diff-c43b0a69f41e5e2a90f01f9983edfdd7) last year. So this problem has been fixed and will be fixed for RStudio users when the when a newer version of `pandoc` is bundled with RStudio.
